specs/airr-schema.yaml

```diff
@@ -1847,19 +1847,38 @@ Rearrangement:
                 subset: data (processed sequence)
                 name: V(D)J germline reference database
 
+# 1-to-n relationship between a CloneSet and the Repertoires it is derived from
+# Question: Do we want to include a direct link to `Subject` in this object?
+CloneSet:
+    discriminator: AIRR
+    type: object
+    properties:
+        clone_set_id:
+            type: string
+            description: Identifier for the clone set object.
+        repertoire_list: #did I do this correctly?
+            type: array
+            description: List of repertoires that were processed to produce clone set
+            items:
+                - $ref: '#/Repertoire'
+        data_processing_id:
+            type: string
+            description: Indentifier for the data processing object
+            $ref: '#/DataProcessing'
+
+
 # A unique clone object that has been constructed from a single clone processing.
-# The information provided here is common among or aggregated across all repertoires
-# in the clone processing.
 Clone:
     discriminator: AIRR
     type: object
     properties:
         clone_id:
             type: string
             description: Identifier for the clone.
-        data_processing_id:
+        clone_set_id:
             type: string
             description: Identifier for the data processing object.
+            $ref: '#/DataProcessing' #is this right?
```

Comment:
@scharch I think the `$ref: '#/<thing>'` is yaml's way of deduplicating something that would be literally copied there. So I think we would not want to use that in cases like this since we want to just keep an ID rather than a copy of the object referred to by the ID, if that makes sense.

Reply:
I am following the `Repertoire` schema here, to be consistent:
https://github.com/airr-community/airr-standards/blob/2ea341e121e7e6e8c2fa886ee6976a94ea82cd82/specs/airr-schema.yaml#L1189
@schristley?

Reply:
Whoops, [looks like this actually comes from Swagger](https://swagger.io/docs/specification/using-ref/). In any case, I think 
> we would not want to use that in cases like this since we want to just keep an ID rather than a copy of the object referred to by the ID, if that makes sense.

still applies.